### DISCUSSION TITLE
Tmc 2209 or 2226 drivers on V400?

### DISCUSSION THREAD

**Discussion by Winky1st** [Guilouz/Klipper-Flsun-Speeder-Pad#109] [Q&A] [answered]
Hi all  and Thank to Cyril for his great work!
Mine it is more a "curiosity"  question as the printer is perfectly working after installing this klipper ver.
why on printer.cfg the drivers are called as tmc2209 wile they should be 2226 (on V400) instead? i guess its kind of same driver as all its working. but better ask. what have i missed ? sorry if its a noob question, i am not a klipper expert.
thank you for helping and wish you a great day

**Comment by dev-zetta** [comment]
My FLSUN V400 was shipped with MKS Robin Nano V2.1 and TMC 2209 drivers.
You have TMC 2226 drivers?

On Thu, May 25, 2023 at 2:30 PM Winky1st ***@***.***> wrote:

> Hi all and Thank to Cyril for his great work!
> Mine it is more a "curiosity" question as the printer is perfectly working
> after installing this klipper ver.
> why on printer.cfg the drivers are called as tmc2209 wile they should be
> 2226 instead? i guess its kind of same driver as all its working. but
> better ask. what have i missed ?
> thank you for helping and wish you a great day
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/discussions/109>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZBWB3XRWW2GYP5D3FIPPTXH5GHPANCNFSM6AAAAAAYOYV2LM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Have a nice day,

Gabriel

**Reply by Winky1st** [reply]
Hallo :)
Actually I dont know, I haven't pulled them off yet. I presumed they were 2226 as by faq page - https://flsun3d.com/pages/v400-faq
I should - and will - investigate then thankyou.
cheers

**Comment by Freddyb888** [comment, marked as answer]
They are the same, only 2226 get less hot. They are more suited for
confined spaces.
